pkg/grafana/grafana.go

```diff
@@ -13,9 +14,50 @@ import (
 )
 
 const (
-	SharedOrgName = "Shared Org"
+	datasourceProxyAccessMode = "proxy"
 )
 
+var SharedOrg = Organization{
+	ID:       1,
+	Name:     "Shared Org",
+	TenantID: "giantswarm",
+}
+
+// We need to use a custom name for now until we can replace the existing datasources.
+var defaultDatasources = []Datasource{
+	{
+		Name:      "Alertmanager olly-op",
+		Type:      "alertmanager",
+		IsDefault: true,
+		URL:       "http://alertmanager-operated.monitoring.svc:9093",
+		Access:    datasourceProxyAccessMode,
+		JSONData: map[string]interface{}{
+			"handleGrafanaManagedAlerts": false,
+			"implementation":             "prometheus",
+		},
+	},
+	{
+		Name:      "Mimir olly-op",
+		Type:      "prometheus",
+		IsDefault: true,
+		URL:       "http://mimir-gateway.mimir.svc/prometheus",
+		Access:    datasourceProxyAccessMode,
+		JSONData: map[string]interface{}{
+			"cacheLevel":     "None",
+			"httpMethod":     "POST",
+			"mimirVersion":   "2.14.0",
+			"prometheusType": "Mimir",
+			"timeInterval":   "60s",
+		},
+	},
+	{
+		Name:   "Loki olly-op",
+		Type:   "loki",
+		URL:    "http://loki-gateway.loki.svc",
+		Access: datasourceProxyAccessMode,
+	},
+}
+
 func CreateOrganization(ctx context.Context, grafanaAPI *client.GrafanaHTTPAPI, organization Organization) (Organization, error) {
```

Comment:
This function signature should rather be `(*Organization, error)` in order to be able to return `nil, err` in case of an error, as we should not return any results when an error occurred.

Reply:
I'll do that change monday but i'm not sure this nitpicking really add any value to this PR that has been delayed for more than a week now 🤔